src/components/Onboarding/Screens/Apps/TokensSettings.js

```diff
@@ -1,21 +1,15 @@
 import React from 'react'
-import { useOnboardingState } from '@providers/Onboarding'
-import Navigation from '../../Navigation'
-
-function TokensSettings({ title }) {
-  const { onBack, onNext, step, steps } = useOnboardingState()
+import { useOnboardingState } from '../../../../providers/Onboarding'
+import TokensSettingsNative from './TokensSettingsNative'
+import TokensSettingsBYOT from './TokensSettingsBYOT'
 
+function TokensSettings() {
+  const { config } = useOnboardingState()
   return (
-    <div>
-      {title}
-      <Navigation
-        backEnabled
-        nextEnabled
-        nextLabel={`Next: ${steps[step + 1].title}`}
-        onBack={onBack}
-        onNext={onNext}
-      />
-    </div>
+    <>
```

Comment:
Is this the way you want the screens to switch? I decided to split the screen into two different because it was getting way too long and the shared code was not that much.